olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.h

```diff
@@ -63,6 +63,9 @@ class PartitionsCacheRepository final {
                        const std::vector<std::string>& partitionIds,
                        const std::string& layer_id);
 
+  bool ClearPartition(const std::string& partitionId,
```

Comment:
Do we need this at all? Can we not just access directly PartitionsCacheRepository from VersionedLayerClientImpl?

Reply:
It's a bit changed now, but still `PartitionsCacheRepository` had new method. Main purpose is to not create cache keys in VersionedLayerClientImpl.